packages/store/internals/src/internal-tokens.ts

```diff
@@ -0,0 +1,15 @@
+import { InjectionToken } from '@angular/core';
+
+/**
+ * @see only use to access internal {@link StateContextFactory} in plugins
+ */
+export const NGXS_INTERNAL_CONTEXT_FACTORY_TOKEN: InjectionToken<string> = new InjectionToken(
+  'NGXS_INTERNAL_CONTEXT_FACTORY_TOKEN'
+);
+
+/**
+ * @see only use to access internal {@link StateFactory} in plugins
+ */
+export const NGXS_INTERNAL_FACTORY_TOKEN: InjectionToken<string> = new InjectionToken(
```

Comment:
So as we decided to change it according the Angular convention:
```suggestion
export const NGXS_STATE_FACTORY = new InjectionToken<unknown>('Internals.StateFactory')
```
And I guess it's not `<string>`, let's leave `unknown` here as it's realistically unknown (because this package doesn't know anything about the real class it references, fix me if I'm wrong).